content/sensu-go/5.17/reference/secrets.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Secrets"
+linkTitle: "Secrets"
+description: "Sensu's secrets management capability allows you to avoid exposing secrets in your Sensu configuration. Read the reference to obtain secrets from one or more external secrets management providers and use sensuctl to manage secrets."
```

Comment:
```suggestion
description: "Sensu's secrets management feature allows you to avoid exposing secrets like usernames, passwords, and access keys in your Sensu configuration. Read the reference to obtain secrets from one or more external secrets providers and use sensuctl to manage secrets."
```